mustermann/lib/mustermann/ast/node.rb

```diff
@@ -91,6 +93,10 @@ def type
         self.class.type
       end
 
+      class << self
+        ruby2_keywords :parse
+      end
```

Comment:
Could you move it just below the declaration of `def self.parse`? Otherwise it's hard to find (in both directions).